xcode/Subconscious/Shared/Components/FirstRun/FirstRunProfileView.swift

```diff
@@ -13,18 +13,34 @@ struct FirstRunProfileView: View {
     @Environment(\.colorScheme) var colorScheme
     
     var body: some View {
+        let did = Did(app.state.sphereIdentity ?? "") ?? Config.default.subconsciousGeistDid
         NavigationStack {
-            VStack {
+            VStack(spacing: AppTheme.padding) {
                 Spacer()
+                StackedGlowingImage(
+                    width: 180,
+                    height: 180
+                ) {
+                    GenerativeProfilePic(
+                        did: did,
+                        size: 180
+                    )
+                }
+                
+                Text("This is your sphere. It stores your notes.")
+                    .foregroundColor(.secondary)
+                Spacer()
+                Text("What should we name it?")
+                        .foregroundColor(.secondary)
                 VStack(alignment: .leading, spacing: AppTheme.unit4) {
                     ValidatedTextField(
-                        placeholder: "nickname",
+                        placeholder: "my-nickname",
                         text: Binding(
                             get: { app.state.nicknameFormFieldValue },
                             send: app.send,
                             tag: AppAction.setNickname
                         ),
-                        caption: "Lowercase letters, numbers and dashes only.",
+                        caption: "This is how others will see you. Lowercase letters, numbers and dashes only.",
```

Comment:
"What should we call you?"

Rationale: it might not be how others see you if they choose to give you a different petname. The function of the nickname is to answer "what should we call you?". It's a default.